### ISSUE TITLE
Implement filters in /compose/status

### ISSUE THREAD

**Issue by ondrejbudai** [osbuild/osbuild-composer#297]
The `/compose/status` route has a filtering feature in lorax:

```
/api/v0/compose/status/<uuids>[?blueprint=<blueprint_name>&status=<compose_status>&type=<compose_type>]
```

We do not currently support this in `osbuild-composer`.

**Comment by jkozol** [comment]
@ondrejbudai this was implemented in osbuild-composer per #229

**Comment by ondrejbudai** [comment]
Oh but the TODO wasn't deleted. :-D Sorry for creating this issue, my bad.
